server/src/main/java/org/apache/iotdb/db/conf/IoTDBConstant.java

```diff
@@ -54,12 +54,10 @@ private IoTDBConstant() {
   public static final String MIN_TIME = "min_time";
   public static final int MIN_SUPPORTED_JDK_VERSION = 8;
 
-  // for cluster, set read consistency level
-  public static final String SET_READ_CONSISTENCY_LEVEL_PATTERN = "set\\s+read.*level.*";
-
-  public static final String SHOW_FLUSH_TASK_INFO = "show\\s+flush\\s+task\\s+info";
-
-  public static final String SHOW_DYNAMIC_PARAMETERS = "show\\s+dynamic\\s+parameters";
+  // show info
+  public static final String ITEM = "                             item";
+  public static final String PARAMETER = "                                          parameter";
```

Comment:
Why put many spaces?

Reply:
Just for the output format. If remove these spaces, its output is as follow:
![image](https://user-images.githubusercontent.com/26211279/68927554-fee9b380-07c2-11ea-9afc-40c4a9e11dbe.png)